src/Aspire.Hosting.Azure/AzureProvisioningResourceExtensions.cs

```diff
@@ -232,21 +232,22 @@ public static ProvisioningParameter AsProvisioningParameter(this EndpointReferen
     /// </param>
     /// <param name="infrastructure">The <see cref="AzureResourceInfrastructure"/> that contains the <see cref="ProvisioningParameter"/>.</param>
     /// <param name="parameterName">The name of the parameter to be assigned.</param>
```

Comment:
Add overloads, don’t just add a new optional parameter, that’s a breaking change